dali/operators/reader/loader/webdataset/tar_utils.cc

```diff
@@ -242,19 +246,20 @@ inline void TarArchive::ParseHeader() {
   readoffset_ = 0;
 }
 
-void TarArchive::Close() {
+std::unique_ptr<FileStream> TarArchive::Close() {
```

Comment:
This is no longer `Close` - more like `Release` or `Detach`.

Reply:
alright